### PR TITLE
move theme from dependency repo

### PR DESCRIPTION
**Pull request by OsmondJiang**
make the dependency repo more clear and clean, add localization convention for theme config so loc user no need to overwrite this config for every locale

### CODE REVIEW

File: src/docfx/localization/LocalizationConvention.cs [hunk 1]
```diff
@@ -50,12 +48,15 @@ public static (string remote, string branch) GetLocalizationRepo(LocalizationMap
 
             var newLocale = mapping == LocalizationMapping.Repository ? $".{locale}" : ".localization";
             var newBranch = bilingual ? $"{branch}-sxs" : branch;
-            var repoName = remote.Split(new char[] { '/', '\\' }).Last();
-            var match = s_repoNameWithLocale.Match(repoName);
-            if (match.Success && match.Groups.Count >= 2 && !string.IsNullOrEmpty(match.Groups[1].Value))
+
+            if (remote.EndsWith($".{defaultLocale}", StringComparison.OrdinalIgnoreCase))
```
Comment (yufeih) [inline src/docfx/localization/LocalizationConvention.cs]: we may always need a reverse lookup from loc -> English to support build, does this affect that design?

Comment (OsmondJiang) [reply src/docfx/localization/LocalizationConvention.cs]: not yet, I would like to do that in another PR, this is only for source to loc

**Comment by OsmondJiang** [comment]
@mingwli, maybe you need change your migrator logic

File: src/docfx/localization/LocalizationConvention.cs (hunk 1 above)
Comment (live1206) [inline src/docfx/localization/LocalizationConvention.cs]: extract this method to a method? duplicated in `GetLocalizationTheme`. Might be more duplication

**Pull request merged**